core/utils/poly-look/src/visualisations/charts/bubble-charts/bubbleCluster/bubbleCluster.js

```diff
@@ -140,18 +148,16 @@ export class BubbleCluster extends Chart {
       .attr("class", "bubble")
       .attr("r", (d) => d.r)
       .style("fill", this._bubbleColor)
-      .style("stroke", "#f7fafc")
+      .style("stroke", this._strokeColor)
       .style("vertical-align", "center")
       .attr("fill-opacity", this._opacity);
   }
 
   _addTextToBubbleGroup(newBubbleGroups) {
     newBubbleGroups
       .append("text")
-      .attr("class", "bubble-value")
-      .text((d) => {
-        return d.r > smallBubblesRadius ? Math.round(d.value) : "";
-      })
+      .attr("class", "bubble-text")
+      .text(this._text)
```

Comment:
@JJ This is what I mean for "HTML text tag equivalent".

Reply:
Right. I think that eventually that default is never used.